### DISCUSSION TITLE
The i3 module should not indicate scrollability

### DISCUSSION THREAD

**Discussion by pvutov** [polybar/polybar#2872] [Ideas]
I have the i3 module enabled with separators, so a sample output is "1|2|3|4".

If I hover the digits, the cursor turns into a pointing finger icon, indicating that I am hovering something clickable. Nice.
If I hover the separators, the cursor turns into a vertical double-sided arrow (something like https://png.pngitem.com/pimgs/s/159-1591691_icon-double-arrow-png-transparent-png.png). I was looking at my cursor randomly turning into a vertical resize indicator, thinking that its telling me that I can resize polybar or something. Just as I was writing up a bug report I got the idea to set ```enable-scroll = false``` in the i3 module and voila, it disappeared. It was a scrollability indicator all along!

If it's not too complicated to do, this scrollability indicator should be turned off IMO. It is misleading. At least on my system it looks exactly the same as the vertical resize indicator. In the context of resizing, I see this thing on a daily basis (for example when I hover the bottom right corner of the text field as I type this github message). I can't think of a time when I've seen the vertical arrow in a scroll-related context. My cursor doesn't turn into a vertical arrow when I hover the scrollbar of my browser, and even scrolling with middle click uses a very different icon.

When someone is hovering the i3 module and their cursor turns into a vertical arrow, I don't think that many will get it as a hint to try the scroll wheel. They're just gonna think that polybar is resizable and/or bugged.

Tried rephrasing so I don't sound cranky but ultimately I'm criticizing a minor feature so coming across as a nitpicker seems unavoidable :)

**Comment by patrick96** [comment]
You're seeing a vertical resize indicator because your [`cursor-scroll`](https://github.com/polybar/polybar/wiki/Configuration#bar-settings) setting is set to `ns-resize` :wink:

If you want to turn it off completely, either delete the setting from your config, set it to nothing (`cursor-scroll = `), or set it to something else (`default` or `pointer`).

**Reply by pvutov** [reply]
Just tried it, thank you. This is exactly the behavior I was hoping for.
